airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -446,10 +445,11 @@ def parse_response(
         """Handle response status"""
         if response.status_code == 200:
             # Ticket ID not included in ticket comments response.
-            # Manually add ticket_id to ticket_comments dict.
+            # Manually add ticket_id and generated_timestamp to ticket_comments dict.
             ticket_id = stream_slice["id"]
+            generated_timestamp = stream_slice["generated_timestamp"]
             result = super().parse_response(response, stream_state=stream_state, stream_slice=stream_slice, **kwargs)
-            enriched_result = map(lambda x: x.update({"ticket_id": ticket_id}) or x, result)
+            enriched_result = map(lambda x: x.update({"ticket_id": ticket_id, "generated_timestamp": generated_timestamp}) or x, result)
             yield from enriched_result
         elif response.status_code == 404:
             ticket_id = stream_slice["id"]
```

Comment:
this function is used by not only `ticket_comments` stream. Did you test updated logic for others (ticket_audits etc)? I think comment records have the key `generated_timestamp`

Reply:
afaik, this parse_response method is part of ticket_comments class. also, comment records doesn't have generated_timestamp key by default.

Reply:
I mean, there's no class that inherit from ticket_comments class

Reply:
You are right, it is mistake. This is custom function's version of the class  TicketComments.